src/t8_cmesh/t8_cmesh_readmshfile.c

```diff
@@ -173,6 +173,63 @@ t8_msh_file_node_compare (const void *node_a, const void *node_b,
   return Node_a->index == Node_b->index;
 }
 
+/* Read an open msh-file and checks whether the MeshFormat-Version is supported by t8code or not. */
+static void
+t8_cmesh_check_version_of_msh_file (FILE * fp)
+{
+
+  char               *line = (char *) malloc (1024);
+  char                first_word[2048] = "\0";
+  T8_ASSERT (fp != NULL);
+  size_t              linen = 1024;
+  int                 retval;
+  int                 max_version_supported = 2;
+  int                 version_number, sub_version_number;
```

Comment:
If you can avoid it, try not  to mix declarations and code in `C`
```suggestion
  size_t              linen = 1024;
  int                 retval;
  int                 max_version_supported = 2;
  int                 version_number, sub_version_number;
 
  T8_ASSERT (fp != NULL);
```